packages/mjml-button/src/index.js

```diff
@@ -1,141 +1,190 @@
-import { MJMLElement, helpers } from 'mjml-core'
-import React, { Component } from 'react'
+import {MJMLElement, helpers} from 'mjml-core'
+import React, {Component} from 'react'
 
-const tagName = 'mj-button'
-const parentTag = ['mj-column', 'mj-hero-content']
-const endingTag = true
+const tagName               = 'mj-button';
```

Comment:
Writing code without semi-colons is like writing a book without any punctuation :(
(But ye, i removed them for consistency)